async-nats/src/jetstream/context.rs

```diff
@@ -52,7 +52,23 @@ impl Context {
         }
     }
 
-    /// Publish a message
+    /// Publish a message to a given subject associated with a stream and returns acknowledgment from
```

Comment:
Publish to a subject, or with a subject?

Reply:
Syntax
PUB <subject> [reply-to] <#bytes>\r\n[payload]
where:
subject: The destination subject to publish to
reply-to: The optional reply inbox subject that subscribers can use to send a response back to the publisher/requestor
#bytes: The payload size in bytes
payload: The message payload data

Thats only reference I found.

Publish vs Publishes - we used the former throughout the docs. I prefer consistency here.